tianshou/env/gym_wrappers.py

```diff
@@ -5,27 +5,24 @@
 class ContinuousToDiscrete(gym.ActionWrapper):
     """Gym environment wrapper to take discrete action in a continous environment.
 
-    param gym.Environment env: gym envirionment with continous action space.
-    param int action_per_branch: number of discrete actions in each dimension
-    of the action space.
+    :param gym.Environment env: gym envirionment with continous action space.
+    :param int action_per_branch: number of discrete actions in each dimension
+        of the action space.
     """
 
     def __init__(self, env: gym.Env, action_per_branch: int) -> None:
         super().__init__(env)
-        self.action_per_branch = action_per_branch
-        low = env.action_space.low  # type: ignore
-        high = env.action_space.high  # type: ignore
-        num_branches = env.action_space.shape[0]  # type: ignore
+        act_space: gym.spaces.Box = env.action_space  # type: ignore
```

Comment:
maybe `assert isinstance(env.action_space, gym.spaces.Box)` here?

Reply:
Yes thats a good solution. With that we won't need the # type: ignore ?

Reply:
yes